src/include/cc.h

```diff
@@ -83,12 +86,34 @@ struct Pcb {
   }
   void rto_advance() { rto_timer++; }
 
+  void shift_right_sack_bitmap() {
+    for (uint8_t i = kSackBitmapSize/64 - 1; i > 0; --i) {
```

Comment:
`-1` should be adjusted if `kSackBitmapSize` is corrected as per previous comment.